jsettlers.main.swing/src/jsettlers/main/swing/foldertree/PathPanel.java

```diff
@@ -0,0 +1,153 @@
+/*******************************************************************************
+ * Copyright (c) 2015 - 2016
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *******************************************************************************/
+package jsettlers.main.swing.foldertree;
+
+import java.awt.BasicStroke;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GradientPaint;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.RenderingHints;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+
+/**
+ * Panel with the path
+ * 
+ * @author Andreas Butti
+ *
+ */
+public class PathPanel extends JPanel {
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * Separator ">" component
+	 */
+	private static class Separator extends JComponent {
+		private static final long serialVersionUID = 1L;
+
+		@Override
+		public void paint(Graphics g1) {
+			Graphics2D g = (Graphics2D) g1;
+			g.setRenderingHint(RenderingHints.KEY_ANTIALIASING, RenderingHints.VALUE_ANTIALIAS_ON);
+			g.setColor(Color.WHITE);
+			g.setStroke(new BasicStroke(2));
+			int w = getWidth();
+			int h = getHeight();
+			g.drawLine(0, 0, w - 1, h / 2);
+			g.drawLine(w - 1, h / 2, 0, h);
+		}
+	}
+
+	/**
+	 * Listener
```

Comment:
Don't add javadoc just that there is a javadoc. In cases like this, where it is obvious, no javadoc is better than this one, because javadocs like this make the code less readable.